src/main/java/com/minecolonies/coremod/util/InventoryFunctions.java

```diff
@@ -56,9 +45,8 @@ public static boolean matchFirstInProvider(
      * @param tester   the function to use for testing slots
      * @return true if it found a stack
      */
-    private static boolean matchFirstInProvider(
-                                                 final ICapabilityProvider provider, @NotNull final Function<ICapabilityProvider, Function<Integer,
-                                                                                                                                            Predicate<ItemStack>>> tester)
+    private static boolean matchFirstInProvider(final ICapabilityProvider provider,
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused private "matchFirstInProvider" method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUnusedPrivateMethod)